doc/architecture/trace_file_formats.adoc

```diff
@@ -0,0 +1,18 @@
+= OSI trace file formats
+
+There are multiple formats for storing multiple serialized OSI messages into one trace file.
+
+*.osi::
+Binary trace file.
+Messages are separated by a length specifications before each message.
```

Comment:
```suggestion
Messages are separated by a length specification before each message.
```

Reply:
Changed.

Reply:
Ok. I'm ok with not changing that. Just noticed the typo.